docs/core/about.md

```diff
@@ -0,0 +1,122 @@
+---
+title: About .NET Core
+description: .NET Core is a modular, high-performance implementation of .NET for creating Windows, Linux, and Mac apps. Learn about .NET Core to get started.
+author: richlander
+ms.author: mairaw
+ms.date: 08/01/2018
+---
+# About .NET Core
+
+.NET Core has the following characteristics:
+
+- **Cross-platform:** Runs on Windows, macOS and Linux [operating systems](https://github.com/dotnet/core/blob/master/os-lifecycle-policy.md).
+- **Consistent across architecures:** Runs your code with the same behavior on multiple architectures, including x64, x86, and ARM.
+- **Command-line tools:**  Includes easy-to-use command-line tools that be used for local development and in continuous-integration scenarios.
+- **Flexible deployment:** Can be included in your app or installed side-by-side user- or machine-wide. Can be used with [Docker containers](docker/index.md).
+- **Compatible:** .NET Core is compatible with .NET Framework, Xamarin and Mono, via [.NET Standard](../standard/net-standard.md).
+- **Open source:** The .NET Core platform is open source, using MIT and Apache 2 licenses. .NET Core is a [.NET Foundation](https://dotnetfoundation.org/) project.
+- **Supported by Microsoft:** .NET Core is supported by Microsoft, per [.NET Core Support](https://www.microsoft.com/net/core/support/).
+
+## Languages
+
+C#, Visual Basic, and F# languages can be used to write applications and libraries for .NET Core. These languages are or can be integrated into your favorite text editors and IDEs, including [Visual Studio](https://visualstudio.microsoft.com/vs/), [Visual Studio Code](https://marketplace.visualstudio.com/items?itemName=ms-vscode.csharp), Sublime Text and Vim. This integration is provided, in part, by the good folks from the [OmniSharp](http://www.omnisharp.net/) and [Ionide](http://ionide.io) projects.
+
+## APIs
+
+.NET Core exposes APIs for many scenarios, a few of which follow:
+
+- Primitive types, such as [string][string] and [int][int].
+- Collections, such as [List<T>][list] and [Dictionary<K,V>][dictionary].
+- Utility types, such as [HttpClient][httpclient], and [FileStream][filestream].
+- Data types, such as [DataSet][dataset], and [DbSet][dbset].
+- High performance types, such as [Vector][vector] and [Pipelines][pipelines].
+
+.NET Core provides compatibility with .NET Framework and Mono APIs by implementing the [.NET Standard](../standard/net-standard.md) specification.
+
+[string]: https://docs.microsoft.com/dotnet/api/system.string?view=netcore-2.1
```

Comment:
Will do.

As a side note, there might be some confusion. That is markdown link syntax. No?

Reply:
My concern was that it would not work with MarkDig (which docs will be using later this year.)  I must have mis-entered the text into [babelmark](https://babelmark.github.io/) when I tested it. It does work there as well.